src/main/java/Entities/Cell.java

```diff
@@ -1,7 +1,7 @@
 package Entities;
 
 abstract class Cell {
-    public String performAction(){
-        return "";
-    }
+
+    public abstract String performAction(Player player, Board board);
```

Comment:
I called the player in my Property PR to currentPlayer and merged my PR to main already. Could you change this to parameter to currentPlayer? This way we will not have any merge conflicts.